From ec3bd363eda7bffaea96c8b5c994593b903a24cf Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 21 Oct 2021 15:41:21 +0200
Subject: [PATCH 0002/1141] ARM: configs: add REBOOT MODE support in
 fragment-02-multiv7_addons

Enable REBOOT MODE support.

Change-Id: I87bfa8d8114da0d589e7ed49840ae49aab354dad
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/316837
Tested-by: Simeon MARIJON <simeon.marijon@st.com>
Reviewed-by: Simeon MARIJON <simeon.marijon@st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/fragment-02-multiv7_addons.config b/arch/arm/configs/fragment-02-multiv7_addons.config
index 981ed4d22f3f..a96f07ec81c5 100644
--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -58,6 +58,7 @@ CONFIG_POWER_RESET_SYSCON=y
 CONFIG_PREEMPT=y
 CONFIG_PREEMPT_COUNT=y
 CONFIG_PROFILING=y
+CONFIG_REBOOT_MODE=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REGULATOR_PROTECTION_CONSUMER=y
 CONFIG_REMOTEPROC=y
@@ -76,6 +77,7 @@ CONFIG_SERIAL_NONSTANDARD=y
 CONFIG_SRAM=y
 CONFIG_STM32_IPCC=y
 CONFIG_STM32_RPROC=y
+CONFIG_SYSCON_REBOOT_MODE=y
 CONFIG_TEE=y
 CONFIG_THERMAL=y
 CONFIG_TMPFS=y
-- 
2.39.2

